pkg/kube/roles.go

```diff
@@ -117,17 +128,17 @@ func UpdateUserRoles(kubeClient kubernetes.Interface, jxClient versioned.Interfa
 		} else {
 			found := false
 			for idx, subject := range envRole.Spec.Subjects {
-				if subject.Kind == userKind && subject.Name == userName {
+				if subject.Kind == userKind && subject.Name == userName && subject.Namespace == adminNs {
```

Comment:
we could have users in multiple namespaces. unlikely but possible - so lets at least check.